en/src/components/SiteNav/styles.module.css

```
@@ -0,0 +1,194 @@
+.drawerOpen {
+  transform: translateX(0);
+}
+
+.drawerScroll {
+  flex: 1;
+  min-height: 0;
+  overflow-y: auto;
+  padding-top: 0.75rem;
+}
+
+.drawerScroll :global(.menu) {
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)